src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, married, partner } = person;
+
+  let partnerInfo;
+
+  if (married) {
+    if (partner.gender === 'female') {
```

Comment:
```suggestion
    if (partner.gender === 'f') {
```
Check what gender value you have. 
Now all person not married! ( check demo link)